model-optimizer/mo/ops/deformable_convolution.py

```diff
@@ -27,4 +27,7 @@ def __init__(self, graph: Graph, attrs: dict):
     def backend_attrs(self):
         # the same attributes as in a regular convolution and additional attributes 'deformable_group', 'group'
         # and 'bilinear_interpolation_pad'
-        return Convolution(self.graph, {}).backend_attrs() + ['deformable_group', 'group', 'bilinear_interpolation_pad']
+        attrs = ['deformable_group', 'group']
+        if self.get_opset() == 'opset8':
+            attrs.append('bilinear_interpolation_pad')
+        return attrs
```

Comment:
How about `Convolution(self.graph, {}).backend_attrs()` ? We you have removed them?

Reply:
oops, my apologies

Reply:
fixed